src/Microsoft.DotNet.Darc/src/Darc/Operations/GatherDropOperation.cs

```diff
@@ -337,6 +358,198 @@ public string GetFileShareLocationForReleaseJson(DownloadedBuild build)
+        {
+            // Core
+
+            // Public
+            { $"{githubRepoPrefix}dotnet/corefx", (coreRepoCategory, "corefx") },
+            { $"{githubRepoPrefix}dotnet/coreclr", (coreRepoCategory, "coreclr") },
+            { $"{githubRepoPrefix}dotnet/core-setup", (coreRepoCategory, "core-setup") },
+            // Internal
+            { $"{azdoRepoPrefix}dotnet-corefx", (coreRepoCategory, "corefx") },
+            { $"{azdoRepoPrefix}dotnet-coreclr", (coreRepoCategory, "coreclr") },
+            { $"{azdoRepoPrefix}dotnet-core-setup", (coreRepoCategory, "core-setup") },
+
+            // ASPNET
+
+            // Public
+            { $"{githubRepoPrefix}dotnet/extensions", (aspnetCategory, "extensions") },
+            { $"{githubRepoPrefix}dotnet/aspnetcore", (aspnetCategory, "aspnetcore") },
+            { $"{githubRepoPrefix}dotnet/aspnetcore-tooling", (aspnetCategory, "aspnetcore-tooling") },
+            { $"{githubRepoPrefix}dotnet/efcore", (aspnetCategory, "efcore") },
+            { $"{githubRepoPrefix}dotnet/ef6", (aspnetCategory, "ef6") },
+            { $"{githubRepoPrefix}dotnet/blazor", (aspnetCategory, "blazor") },
+            // Internal
+            { $"{azdoRepoPrefix}dotnet-extensions", (aspnetCategory, "extensions") },
+            { $"{azdoRepoPrefix}dotnet-aspnetcore", (aspnetCategory, "aspnetcore") },
+            { $"{azdoRepoPrefix}dotnet-aspnetcore-tooling", (aspnetCategory, "aspnetcore-tooling") },
+            { $"{azdoRepoPrefix}dotnet-efcore", (aspnetCategory, "efcore") },
+            { $"{azdoRepoPrefix}dotnet-ef6", (aspnetCategory, "ef6") },
+            { $"{azdoRepoPrefix}dotnet-blazor", (aspnetCategory, "blazor") },
+
+            // WCF
+
+            // Public
+            { $"{githubRepoPrefix}dotnet/wcf", (wcfCategory, "wcf") },
+            // Internal
+            { $"{azdoRepoPrefix}dotnet-wcf", (wcfCategory, "wcf") },
+        };
+
+        private static readonly List<string> doNotListSymbolPackageFilenamePrefixes = new List<string>()
+        {
+            // Do not include targeting pack symbol packages: https://github.com/dotnet/core-setup/issues/8310
+            // They shouldn't have PDBs anyway, but due to tool behavior plus an artifact issue,
+            // they do have PDB and publishing them breaks PDB conversion later on.
+            "Microsoft.NETCore.App.Ref.",
+            "Microsoft.NETCore.App.Internal.",
+            "Microsoft.WindowsDesktop.App.Ref."
+        };
+
+        const string identitiesDirectoryName = "identities";
+        const string nupkgDirectoryName = "nupkgs";
+        const string sympkgsDirectoryName = "sympkgs";
+        const string allNupkgsFileName = "nupkgs-all-just-for-reference.txt";
+        const string nupkgsFileNamePrefix = "nupkgs-";
+        const string sympkgsFileName = "sympkgs-all.txt";
+        const string packagesSubDirs = "packages";
```

Comment:
I don't understand why this is plural, vs `packagesSubDir`.

Reply:
I think this is what's under, for example, /nuget/nupkgs/dotnet-core-setup

Reply:
Should just be subdir